.gitignore

```diff
@@ -24,3 +24,4 @@ config.codekit
 /node_modules/
 composer.phar
 composer-dev.*
+/web/assets/images/fallback_override.png
```

Comment:
Maybe throw in a comment explaining why we're ignoring this?

Reply:
Hm. It comes from upstream, so I'd rather not I think :)
https://github.com/os2display/admin/commit/02b6fbfdf6ff0917e7c63e08eb0e2d380d06b24d